config/sync/config.settings.yml

```diff
@@ -1,5 +1,10 @@
 custom:
   name: sandipta
   email: null
-  number: '9432406679123'
+  number: '9432406679'
   gender: '0'
+Name: null
+'Phone Number': '8583942894'
+Email: sandipta.sardar@gmail.com
+Gender: male
+Subscription: "I don't feel like"
```

Comment:
1/ keys should be small letters
2/ keys should be snake cased

Reply:
This is of the last ticket, changes are already being done on https://github.com/sandipta18/Drupal-Backend/pull/4